crates/next-core/src/page_source.rs

```diff
@@ -192,23 +193,22 @@ pub async fn create_page_source(
         .collect(),
     );
 
+    let client_context: AssetContextVc = ModuleAssetContextVc::new(
+        transitions,
```

Comment:
All AssetContext get passed the same transitions. So `context_context` is equal to the client context created by the `server_context` + client transition

Reply:
This is definitely an easy to make mistake.